CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # TAMR AWS EMR module
 
+## v5.1.0 - July 6th, 2021
```

Comment:
Sorry, late review but we should also bump [VERSION](https://github.com/Datatamer/terraform-aws-emr/blob/master/VERSION).

Reply:
My bad. Opened [PR#48](https://github.com/Datatamer/terraform-aws-emr/pull/48) to fix this.